src/services/utilities.ts

```diff
@@ -2154,6 +2155,16 @@ export function isStringOrRegularExpressionOrTemplateLiteral(kind: SyntaxKind):
     return false;
 }
 
+export const isStringAndEmptyAnonymousObjectIntersection = (type: Type) => {

```

Comment:
Can you make this a function declaration and mark it as `@internal`? Sorry, I didn’t catch that that’s what the API baseline change is. You’ll need to rerun the Public API test which will essentially undo the addition from the last commit. My bad.

Reply:
To be honest I thought its okay to introduce methods with that most probably won't change because of obvious name and implementation, but thanks for letting me know!

Reply:
```suggestion
export function isStringAndEmptyAnonymousObjectIntersection(type: Type) {
```

Reply:
Went ahead and committed this to save some back and forth, hope you don’t mind